Project Euler/Problem 16/sol1.py

```diff
@@ -0,0 +1,15 @@
+power = int(input("Enter the power of 2: "))
+num = 2**power
+
+string_num = str(num)
+
+list_num = list(string_num)
+
+sum_of_num = 0
+
+print("2 ^",power,"=",num)
+
+for i in list_num:
+    sum_of_num += int(i)
+
+print("Sum of the digits are:",sum_of_num)
```

Comment:
https://docs.python.org/3/library/functions.html#sum

print("Sum of the digits are:", sum(int(i) for i in string_num))